packages/eas-cli/src/commands/build/submit.ts

```diff
@@ -75,13 +75,31 @@ export default class BuildSubmit extends Command {
     'apple-id': flags.string({
       description: 'Your Apple ID username (you can also set EXPO_APPLE_ID env variable)',
     }),
-    'apple-app-specific-password': flags.string({
-      description:
-        'Your Apple ID app-specific password. You can also set EXPO_APPLE_APP_SPECIFIC_PASSWORD env variable.',
-    }),
     'app-apple-id': flags.string({
       description: 'App Store Connect unique application Apple ID number.',
     }),
+    'apple-team-id': flags.string({
+      description: 'Your Apple Developer Team ID',
+    }),
+    'app-name': flags.string({
+      description:
+        "The name of your app as it will appear on the App Store, this can't be longer than 30 characters (default: expo.name from app.json)",
+    }),
+    'bundle-identifier': flags.string({
+      description: 'Your iOS Bundle Identifier (default: expo.ios.bundleIdentifier from app.json)',
+    }),
+    sku: flags.string({
+      description:
+        'An unique ID for your app that is not visible on the App Store, will be generated unless provided',
+    }),
+    language: flags.string({
+      description: 'Primary language (e.g. English, German, ...)',
+      default: 'English',
+    }),
+    'company-name': flags.string({
+      description:
+        'The name of your company, needed only for the first upload of any app to App Store',
+    }),
```

Comment:
It should be clear for user what flags are platform specific, in other cases it's clear based on the description, but not here

We should prefix all options with platform or change description for those two to make it clear that it's only for ios

Reply:
There are plans for custom help for this command, at least for groupping flags by platform. I'm aware that now they're all mixed and nobody knows which flag is used for which platform, and I won't agree that in other cases it's clear ;) I've been talking with @fson about this.